units/pod_creation.go

```diff
@@ -125,11 +124,6 @@ func (j *podCreationJob) Run(ctx context.Context) {
 			return
 		}
 
-		if err := podDef.UpdateLastAccessed(); err != nil {
```

Comment:
Why are we now unconditionally updating `LastAccessed` rather than only when the pod is initializing?

Reply:
The GitHub PR view is buggy and isn't showing this comment when I click on the files changed... It's also having some difficulty letting me submit this comment. 😭 

This logic is equivalent to what it was before. I just moved updating the access time into `checkForPodDefinition`.

Reply:
Ah, I see `checkForPodDefinition` returns early if the pod isnt initializing. Sorry! Never mind